docs/backend-requests/versioning/overview.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Versioning Overview
+description: A brief guide on how versioning works in Clerk.
+---
+
+# Versioning
+
+When backwards incompatible changes are necessary for an API, new versions are released to prevent disruption to existing applications. These versions are identified by their release date, such as 2021-02-05. Both the frontend and backend APIs follow the same version.
+
+A complete list of all the available versions with their changes can be found on the [API versions](/docs/backend-requests/versioning/available-versions) page.
+
+## What constitutes a breaking change
+
+A breaking change is any modification that would require users to alter their existing setups. These include:
+
+- **Change in property type**: Altering the data type of a property in either requests or responses.
+- **Removal of a property**: Deleting a property or parameter from a request or response.
+- **Change in property name**: Renaming a property in request or response payloads.
+- **Backwards incompatible endpoint changes**: Implementing changes in an endpoint's functionality that older client versions cannot support. For instance, adding a new status to the sign-in process.
+- **Endpoint removal**: Discontinuing access to an existing endpoint, which remains accessible in earlier versions.
+
+## When using an SDK
+
+Each Clerk SDK version corresponds to a specific API version, so by updating the SDK, you're also updating to the latest compatible API version.
+
+### Choosing an API version
+
+When making direct API calls to an endpoint, there are three options to specify the version:
+
+1. **Query parameter**: Set the `__clerk_api_version` query parameter in your request URL.
+2. **Clerk-API-Version header**: Include a `Clerk-API-Version` header in your requests.
+3. **Dashboard setting**: Set a default version in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys). This version will be used if one is not explicitly specified.
```

Comment:
I don't see where you can set the default version on the API Keys page - did you link to the right place?

Reply:
This will be the place but its not yet available. It will be introduced when the first new (second) version is released 😅 After talking with @kostaspt we decided to remove the section about the dashboard until it is ready so we can merge the docs now.